src/features/views/components/ViewDataTable/columnTypes/LocalPersonColumnType.tsx

```diff
@@ -42,20 +41,19 @@ export default class LocalPersonColumnType
       headerAlign: 'center',
 
       renderCell: (params: GridRenderCellParams) => {
-        return <ZUIPersonGridCell person={params.row[params.field]} />;
+        return <ZUIPersonGridCell person={params.value} />;
       },
       renderEditCell: (params: GridRenderEditCellParams) => {
-        return (
-          <EditCell
-            cell={params.row[params.field]}
-            column={col}
-            row={params.row}
-          />
-        );
+        return <EditCell cell={params.value} column={col} row={params.row} />;
       },
-      valueGetter: (params: GridValueGetterParams) => {
-        const cell = params.row[params.field];
-        return this.cellToString(cell);
+      sortComparator: (v1, v2) => {
+        if (!v1) {
+          return 1;
+        }
+        if (!v2) {
+          return -1;
+        }
+        return v1.first_name.localeCompare(v2.first_name);
```

Comment:
This sorts based on first name, but what happens if two people have the same first name? Those people will end up in "random" order. I think you should also sort on last name. You can do that in one of two ways:

Option 1: Check the return value from `v1.first_name.localeCompare(v2.first_name)` and if it's zero, return the same but for last name instead, like so:

```tsx
return v1.first_name.localeCompare(v2.first_name) || v1.last_name.localeCompare(v2.last_name);
```

Or, you can create a full name variable and compare those instead:

```tsx
const name1 = `${v1.first_name} ${v1.last_name}`;
const name2 = `${v2.first_name} ${v2.last_name}`;
return name1.localeCompare(name2);
```

Reply:
Ah yeah I totally forgot about the last name! My bad!